scripts/notify/index.js

```diff
@@ -14,10 +14,10 @@ const verbs = [
 	'hit the registry!',
 	'to share!',
 	'for you!',
-	'for y’all! 🤠',
-	'comin’ your way!',
-	'comin’ atcha!',
-	'comin’ in hot!',
+	'for y'all! 🤠',
```

Comment:
💀

Reply:
It appears you forgot to escape one of the quotes.